tests/test_main.py

```diff
@@ -46,7 +46,7 @@ def test_shop_trip_output(monkeypatch):
 Thanks, Alex, for your purchase!
 You have bought: 
 2 milks for 4 dollars
-2 breads for 3.0 dollars
+2 breads for 3 dollars
```

Comment:
why did you change it???

Reply:
you can't change test data

Reply:
How can I get the desired result if in one case one result is displayed and in the other another.
<img width="279" alt="image" src="https://github.com/mate-academy/py-shop-trip/assets/60896665/cb62fb17-0ea5-4618-863c-3042fc372d82">